ledger/participant-state/kvutils/tools/README.md

```diff
@@ -24,11 +24,13 @@ Run the tool using Bazel:
 The following options are supported:
 
     --help
-    --skip-byte-comparison  Skips the byte-for-byte comparison. Useful when comparing behavior across versions.
-    --sort-write-set        Sorts the computed write set. Older exports sorted before writing. Newer versions order them intentionally.
-    --index-only            Run only the indexing step of the integrity checker (useful to benchmark the indexer).
-    --jdbc-url              External JDBC url (useful for running against PostgreSQL).
-    --report-metrics          Print all registered metrics.
+    --skip-byte-comparison      Skips the byte-for-byte comparison. Useful when comparing behavior across versions.
+    --sort-write-set            Sorts the computed write set. Older exports sorted before writing. Newer versions order them intentionally.
+    --index-only                Run only the indexing step of the integrity checker (useful to benchmark the indexer).
+    --jdbc-url                  External JDBC url (useful for running against PostgreSQL).
+    --report-metrics            Print all registered metrics.
+    --expected-updates-output   The output path for expected updates. Useful for debugging. It might be worth sorting the output files and using a diff tool.
```

Comment:
Does it make sense to use these separately? I.e., if the user entered a base path or file name prefix we could output the updates as `<prefix>.actual.txt` and `<prefix>.expected.txt`.

Reply:
I think it would make sense if you'd just want to see one update set, maybe it's not worth eliminating this use case.

Reply:
OK